### ISSUE TITLE
Build fail with error cannot find -l:librocksdb.a

### ISSUE THREAD

**Issue by cydrain** [milvus-io/milvus#6940]
#### Steps/Code to reproduce:
```
commit fb5ca43621e09dbd0656053129f4bce2cadad153
Author: milvus-ci-robot <87847967+milvus-ci-robot@users.noreply.github.com>
Date:   Sun Aug 1 17:21:22 2021 +0800
```
1. clone Milvus master branch
2. make all

#### Expected result:
build pass

#### Actual results:
```
warning: go env -w CGO_CFLAGS=... does not override conflicting OS environment variable
# runtime/cgo
/usr/bin/ld: cannot find -l:librocksdb.a
collect2: error: ld returned 1 exit status
Makefile:100: recipe for target 'build-cpp' failed
make: *** [build-cpp] Error 2
```

**Comment by xiaofan-luan** [comment]
hit similar issue, need to invitigate on it

**Comment by cydrain** [comment]
this issue has been fixed by #7200

**Comment by XuanYang-cn** [comment]
> this issue has been fixed by #7200

Reproduced on MacOS, investigating now

**Comment by XuanYang-cn** [comment]
```
warning: go env -w CGO_CFLAGS=... does not override conflicting OS environment variable
```
This is the problem, next time try unseting OS environment variable for CGO_CFLAGS

Feel free to reopen this issue if anything related.
